h2o-core/src/main/java/water/parser/ParseSetup.java

```diff
@@ -42,6 +45,9 @@
   String[][] _na_strings;       // Strings for NA in a given column
   String[][] _data;           // First few rows of parsed/tokenized data
 
+  int tentativeNumLines = -1;
+  int dataOffset = -1;
```

Comment:
I'll comment out tentativeNumLines, but dataOffset is needed for evaluating chunkSize, see line 281 of ParseSetup. I do not see any other way to fix chunkSize. Do you?
In any case, I'm planning to rewrite this stuff later on. But right now the value is required. It will be required in other parsers later on. You don't mean I should, while fixing this specific bug, go ahead and start changing other parsers? All in due time. Trust me.